policy/modules/system/systemd.te

```diff
@@ -479,6 +479,7 @@ seutil_search_default_contexts(systemd_coredump_t)
 allow systemd_generator_t self:fifo_file rw_fifo_file_perms;
 allow systemd_generator_t self:capability { dac_override sys_admin sys_resource };
 allow systemd_generator_t self:process { getcap getsched setfscreate signal };
+modutils_load_kernel_modules(systemd_generator_t)
```

Comment:
This should use the existing `modutils_domtrans()` instead so the existing kmod domain is used.

Reply:
Thanks for your review. Changed to kmod.